components/prism-ruby.js

```diff
@@ -19,3 +19,17 @@ Prism.languages.insertBefore('ruby', 'keyword', {
 	'variable': /[@$]+\b[a-zA-Z_][a-zA-Z_0-9]*[?!]?\b/,
 	'symbol': /:\b[a-zA-Z_][a-zA-Z_0-9]*[?!]?\b/
 });
+
+Prism.languages.insertBefore('ruby', 'comment', {
+        'interpolation': {
+        	pattern: /#\{[^}]+\}/,
+        	inside: {
+        		'delimiter': {
+        			pattern: /^#\{|\}$/,
+        			alias: 'punctuation
```

Comment:
There is a quote missing, no?

Also, could you please add some examples of string interpolations in the file `examples/prism-ruby.html`?